docfiles/head.html

```diff
@@ -1,8 +1,8 @@
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
 <meta name="viewport" content="width=device-width,height=device-height,user-scalable=no,initial-scale=1.0,maximum-scale=1.0,minimum-scale=1.0">
 
-<link rel="stylesheet" href="/doccdn/semantic.css" />
-<link rel="stylesheet" href="/doccdn/blockly.css" />
+<link rel="stylesheet" href="/blb/semantic.css" />
+<link rel="stylesheet" href="/blb/blockly.css" />
```

Comment:
I don't think that `/blb/` paths are currently expanded in docs. I think the issue is that it's unclear against which version of the `-built` repo they should be expanded.